frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js

```diff
@@ -237,6 +242,9 @@ class DatasetActionView extends React.PureComponent<Props, State> {
                   to={`/datasets/${dataset.owningOrganization}/${dataset.name}/edit`}
                   title="Open Dataset Settings"
                   disabled={isReloading}
+                  onClick={() =>
+                    sendAnalyticsEvent("open_dataset_settings", { datasetName: dataset.name })
```

Comment:
I think, it would be better to trigger this event within `componentDidMount` of the settings view. That way, we also catch this event, when another settings link is used (e.g., within the dataset-info-tab of the tracing view).